fhir-bulkimportexport-webapp/liberty-config/server.xml

```diff
@@ -0,0 +1,203 @@
+         for nonsensical queries like requests for a JSP page that doesn't exist.
+    -->
+    <webContainer disableXPoweredBy="true" displaytextwhennoerrorpagedefined="Unexpected request/response. Please check the URL and try again."/>
+
+    <!-- FHIR Server's keystore and truststore configuration -->
+    <ssl clientAuthenticationSupported="true" id="defaultSSLConfig" keyStoreRef="defaultKeyStore" trustStoreRef="defaultTrustStore"/>
+    <keyStore id="defaultKeyStore" location="${server.config.dir}/resources/security/fhirKeystore.jks" password="change-password"/> 
+    <keyStore id="defaultTrustStore" location="${server.config.dir}/resources/security/fhirTruststore.jks" password="change-password"/>
+    <webAppSecurity allowFailOverToBasicAuth="true" singleSignonEnabled="false"/>
+
+    <!-- Define a basic user registry with a few users. -->
+    <basicRegistry id="basic" realm="BasicRealm"> 
+        <user name="fhiruser" password="change-password"/>
+        <user name="fhiradmin" password="change-password"/>
+        <group name="FHIRUsers">
+            <member name="fhiruser"/>
+        </group>
+        <group name="clientAdministrator">
+            <member name="fhiruser"/>
+        </group>
+    </basicRegistry>
+    
+    <authorization-roles id="com.ibm.ws.batch">
+	<security-role name="batchAdmin">	
+		<user name="fhiradmin"/>	
+	</security-role>
+	<security-role name="batchSubmitter">
+		<user name="fhiruser"/>
+	</security-role>
+	<security-role name="batchMonitor">
+		<user name="fhiradmin"/>
+		<user name="fhiruser"/>
+	</security-role>
+</authorization-roles>
+
+    <!--    
+    <openidConnectProvider id="oidc-provider" oauthProviderRef="oauth2-provider"/> 
+    
+    <oauth-roles>
+        <authenticated>
+            <special-subject type="ALL_AUTHENTICATED_USERS"/>  
+        </authenticated>
+        <clientManager>
+            <group name="clientAdministrator" />
+            <user name="fhiruser" />
+        </clientManager>
+    </oauth-roles>
+    
+    <oauthProvider id="oauth2-provider" oauthOnly="false">
+        <grantType>client_credentials</grantType>
+        <databaseStore dataSourceRef="OAuthDataSource" />
+    </oauthProvider>
+    -->
+    
+    <!-- Enable CORS (Cross Origin Resource Sharing) -->
+    <!--  
+    <cors allowCredentials="true" allowedMethods="GET, PUT, POST"
+        allowedHeaders="Content-Type, Authorization, Accept, Origin"
+        allowedOrigins="*" domain="/fhir-server/api/v4" maxAge="3600" />
+    -->    
+    
+    <!-- Uncomment this section to configure a remote Open ID Connect Provider that overrides the oauthProvider configured above. 
+         Update fhir-server-config.json to publish the remote provider OAuth URLs. -->
+    <!-- 
+    <openidConnectClient authorizationEndpointUrl="https://authorize-dstu2.smarthealthit.org/authorize" 
+                    clientId="c6c17d9e-8633-440f-a220-4d077acce0d1" 
```

Comment:
please remove 93/94/95/96

Reply:
good catch, I just searched password and missed these. done